server/index.js

```diff
@@ -0,0 +1,25 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var dummyData = require('./dummyData/dummyData');
+var company = require('./company/index');
+var app = express();
+var router = express.Router();
+
+app.set('port', (process.env.PORT || 8080));
+app.use(bodyParser.json());
+app.use('/company', company);
+// app.use(express.static(path.join(__dirname + '../client/dist')));
+
+router.get('/', function(req, res) {
```

Comment:
app.get   vs   router.get
Look into the which one specifically does what, would be very interested in understanding that distinction